lib/model/katex.dart

```diff
@@ -640,10 +641,10 @@ class _KatexParser {
         _hasError = true;
       }
     }
-    // Currently, we expect `top` to only be inside a vlist, and
-    // we handle that case separately above.
-    if (styles.topEm != null) {
-      throw _KatexHtmlParseError('unsupported inline CSS property: top');
+    // Currently, we expect `top` to be inside a vlist (which we handle it
+    // separately above), and if it's along with a `position: relative`.
+    if (styles.topEm != null && styles.position != KatexSpanPosition.relative) {
+      throw _KatexHtmlParseError('unsupported inline CSS property "top", when "position: ${styles.position}"');
```

Comment:
This comment doesn't parse for me.

I think the vlist part is beside the point now: we do expect a `top` property, but only when accompanied by `position: relative`.

And just saying that isn't useful as a comment, because it repeats what the code in the condition already says. The thing that is useful to say in a comment, though, is *why*. So:
```suggestion
    if (styles.topEm != null && styles.position != KatexSpanPosition.relative) {
      // The meaning of `top` would be different without `position: relative`.
      throw _KatexHtmlParseError(
        'unsupported inline CSS property "top" given "position: ${styles.position}"');
```

I.e.,
> The meaning of `top` would be different without `position: relative`.

As MDN says:
https://developer.mozilla.org/en-US/docs/Web/CSS/top
> The effect of top depends on how the element is positioned (i.e., the value of the [position](https://developer.mozilla.org/en-US/docs/Web/CSS/position) property):
> [… and then a list of 4 quite different meanings for the 5 possible values of `position`.]